internal/git/git.go

```diff
@@ -150,7 +150,7 @@ type Commit struct {
 }
 
 func LatestCommit(ref string) (*Commit, error) {
-	logCmd := GitCommand("show", "-s", "--format='%h %s'", ref)
+	logCmd := GitCommand("show", "-s", "--format=%h %s", ref)
```

Comment:
Yea I thought this too but Go seems to format it correctly and escape the space. I have tested on my machine but that is linux so possible it could cause issues elsewhere. 

Another easy fix would be to just strip the quotes from the output. Let me see if I can write a test to confirm it works.

Reply:
I'm not particularly proficient in writing tests in go so wasn't able to get a worthwhile usecase. I see there is a test git repo set up to run tests against which may allow this to be validated but I tested it against a test repo i have and can confirm it certainly works for me, again on linux. 
![2020-11-16_190653](https://user-images.githubusercontent.com/1794071/99296913-68215e00-283f-11eb-985b-494d2d309780.png)

![2020-11-16_190811](https://user-images.githubusercontent.com/1794071/99296910-6788c780-283f-11eb-843c-d51d6d569c61.png)
![2020-11-16_190815](https://user-images.githubusercontent.com/1794071/99296906-66f03100-283f-11eb-9dac-4bc7eb7afa36.png)

Reply:
Great! My only concern is if it will work on windows

Reply:
For the test case, there's a test for mr_create that tests the autofill as well. You could change this line:
https://github.com/profclems/glab/blob/1666742ac2b17b12a3288b13f619e87dd6208954/commands/mr/create/mr_create_test.go#L130

to be:
```go
cmdtest.Eq(t, out, "!1 Update somefile.txt")
```

This will expect the line to be exactly equal to `!1 Update somefile.txt`. So if the output is `!1 Update somefile.txt'` the test will fail accordingly

Reply:
Run the test with `make test`

Reply:
Nice, thanks. I amended the test. Ended up sticking with the `assert.Contains` as `out` had a bunch of other lines in there so seemed cleaner than `strings.Split(out)[1]` etc. 

I'll open an issue but currently the tests partially fail on my machine, it looks like this is to do with the fact some are looking explicitly for `profclems/glab` in the git remote which for my fork doesn't exist, and also for the `TestNewCmdCreate_autofill` test, it eems to try and push the branch, which again fails.

Reply:
Yhyhyh... You're right. I almost forgot that. I need to fix that.

By the way, the test passed on the CI.

Reply:
I have fixed the test issue with #317 

I have delayed merging this because I am unsure whether it will work on windows or not. I am adding windows test to the CI with #314

Reply:
Nice one

Reply:
This works on windows as well. Thanks